gossip/src/ping_pong.rs

```diff
@@ -153,9 +155,10 @@ impl Signable for Pong {
 }
 
 impl PingCache {
-    pub fn new(ttl: Duration, cap: usize) -> Self {
+    pub fn new(ttl: Duration, rate_limit_delay: Duration, cap: usize) -> Self {
         Self {
             ttl,
```

Comment:
probably would be good to put an `assert` here that the `rate_limit_delay` is sufficiently smaller than `ttl`.

Reply:
added an `assert` for sanity checking